scripts/SMCodeGen/dataModels/ctxtManagerAppModel.json

```diff
@@ -486,9 +506,45 @@
                                                                 "Cardinality": "15"
                                                         }
                                                         ]
+					},
+					{
+						"BlockName": "MmeSmCreateBearerProcCtxt",
+						"BlockType": "Temporary",
+						"Parent": "MmeProcedureCtxt",
+						"PoolSize": "8000",
+						"Cardinality": "1",
+						"Data": [
+							{
+								"Name": "createBearerReqEMsg",
+								"Type": "std::shared_ptr<cmn::EventMessage>"
+							},
+							{
+								"Name":"bearerStatus",
+								"Type": "std::list<BearerCtxtCBResp>"
+							}
+						]
+					},
+					{
+						"BlockName": "SmDedActProcCtxt",
+						"BlockType": "Temporary",
+						"Parent": "MmeProcedureCtxt",
+						"PoolSize": "8000",
```

Comment:
What does this 8000 mean? Simultaneous 8000 procedures can be supported ? 
Do we allocate these pool elements at start ?
Should we keep this limit down, Do we expect these many dedicated bearer creation simultaneously ?

Reply:
Yes. Currently, we are pre-allocating memory for 8000 contexts and these are reserved at the startup. Maybe, we can change mem pool code so that we allocate mem blocks in increments. i.e.. max 8000 blocks allocated in increments of 1000 (dynamic). When 1000 blocks are used up, we can allocate the next 1000 set of blocks..till it reaches 8000)
Can take up as next enhancement item.